macosx/mac-joypad.mm

```diff
@@ -177,7 +152,7 @@ void gamepadAction(void *inContext, IOReturn inResult, void *inSender, IOHIDValu
         pthread_mutex_unlock(&keyLock);
         if (info.min != info.max)
         {
-            if (inputStruct.value <= info.min || inputStruct.value >= info.max)
+            if (inputStruct.value >= info.min && inputStruct.value <= info.max)
```

Comment:
Ugh, actually I see why this code was like this now... will need a bit of work/thinking to fix the breakage.